package.json

```diff
@@ -40,7 +40,8 @@
     "prop-types": "15.7.2",
     "react": "16.8.6",
     "react-addons-pure-render-mixin": "15.6.2",
-    "react-bootstrap": "0.32.4",
+    "react-bootstrap": "github:mattermost/react-bootstrap#dcbd68da4d32c1c4b86e8f2942bc69bc2ebed550",
+    "react-overlays": "^0.8.0",
```

Comment:
Adding `react-overlays` on purpose? If so can we remove the `^` so it doesn't auto update

Reply:
Yup, we were importing `react-overlays` directly from the code but bringing it in as a transient dependency of `react-bootstrap`.